docs/endpoints/post-token-refresh.md

```diff
@@ -19,9 +19,8 @@ Used by: This endpoint is used mainly by publishers.
 Here's what you need to know about this endpoint:
 
 - No encryption is required for token refresh requests.
-- Responses are encrypted only if the HTTP status code is 200. Otherwise, responses are not encrypted.
-- To decrypt responses, the `refresh_response_key` value returned in the [POST /token/generate](post-token-generate.md) or `POST /token/refresh` response from which the refresh token in the request is returned.
-- If you send a refresh token from a v1 `token/generate` response in the request, the response will not be encrypted.
+- If the request is successful, with an HTTP status code of 200 and a new UID2 token or opt-out information returned, the response is encrypted. Error responses are not encrypted. [**GWH/SW_01 opt-out is a 200 response but I'm not sure whether it's encrypted or not. It's mentioned 3 times in this doc. Could we clarify please whether an opt-out response is encrypted?**]
+- To decrypt responses, the `refresh_response_key` value returned in the [POST /token/generate](post-token-generate.md) or `POST /token/refresh` response from which the refresh token in the request is returned. {**GWH/SW_02. Please help me understand this line.**}
```

Comment:
So basically refresh_response_key is the similar concept as "client secret" for token/generate call, that is used to encrypt/decrypt the successful response in token/refresh. So when token/generate creates a refresh token response, it has the first refresh_response_key to be used to encrypt the response of next token/refresh call and then each token/refresh response will a new refresh_response_key for the next subsequent token/refresh call response decryption and so on. 


@jon8787  please confirm if what i said makes sense about the purpose of refresh_response_key

Reply:
That's correct. The reason we have refresh_response_key is because we can't use the initial client secret to decrypt this. That's because decrypting /token/refresh responses happens in the user's browser, so any secret would've been visible in the JavaScript. 

"If the request is successful, with an HTTP status code of 200 and a new UID2 token or opt-out information returned, the response is encrypted." Should we split this into 2 sentences?